checkov/terraform/checks/resource/ncp/LBListenerUsingHTTPS.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class LBListenerUsingHTTPS(BaseResourceCheck):
```

Comment:
@pj991207 This should be a `BaseResourceValueCheck` inheritance IMO

Reply:
Thanks I reflect your advice